src/styles/blocks/stars.css

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  gap: 17px;
+}
+
+.card__stars {
+  margin-top: 16px;
+}
```

Comment:
you should try
  display: flex;
  justify-content: space-between;
  align-items: center;